denops/@ddu-kinds/lsp_codeAction.ts

```diff
@@ -66,8 +66,8 @@ async function ensureAction(
         "codeAction/resolve",
         item.data,
         action.context.bufNr,
-      ) as CodeAction;
-      action.edit = resolvedCodeAction.edit;
+      ) as CodeAction | undefined;
```

Comment:
may be `| null`

Reply:
In my case, both null | undefined seemed to work, so I set it to undefined for the purpose of matching the surrounding code.
Change to null.

Reply:
The specification of lspRequest is that it should return null, not undefined. Thanks for the response.